UI/Navigation/src/Navigation/Presentation/ModalDialog/ComplexFlyoutPage.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Flyout x:Class="Navigation.Presentation.ComplexFlyoutPage"
```

Comment:
Can we try to get the Complex Flyout better positioned?

## Windows
![image](https://github.com/user-attachments/assets/f0ef2086-6fd7-441b-9041-dfd02db7a9c5)

## Others
![image](https://github.com/user-attachments/assets/d5b6645f-219c-46d7-80c6-77fb017d2542)